hxl_proxy/util.py

```diff
@@ -59,8 +62,47 @@ def skip_cache_p ():
 # Input options
 ########################################################################
 
+def make_input (raw_source, input_options=None):
+    """ Wrapper for libhxl make_input call.
+    We want to make sure all exception are being caught and no trace is being made.
+    We also want to check the external destination against our allowed domain list
+    @returns: False if some error occured, the input object otherwise
```

Comment:
@davidmegginson, I think that here is actually wrong doc. It is returning an exception and logs the error, it does not return bool False. My mistake, sorry. Maybe you can sneak this fix during your work as well.

Reply:
Apologies -- I removed the returning false but didn't update the docstring. I'll fix that. (Catching an exception here isn't useful, because a lot of the input work is lazy, and happens later on.)

Reply:
It should throw an exception if the destination is not in the allowed list. Same if the underlying code throws an exception it should catch it, log it and throw it as well.